packages/idafree.vm/tools/chocolateyinstall.ps1

```diff
@@ -31,15 +31,22 @@ try {
   }
 
   $menuIcon = Join-Path $toolDir "ida.ico" -Resolve
-  # Run a Powershell script to open with last IDA Pro version which is likely installed after the IDA free package.
-  # It takes slightly longer than using an static path but it works after installing IDA Pro and every time you update it.
-  # The "-WindowStyle hidden" still shows the Powershell Window briefly: https://github.com/PowerShell/PowerShell/issues/3028
-  # We could use the run-hidden wrapper, which won't display the Window but is likely slightly slower.
-  $script = "`$idaExecutable = Get-Item '$Env:programfiles\IDA Pro *\ida.exe' | Select-Object -Last 1; if (!`$idaExecutable) { `$idaExecutable = '$executablePath' }; & `$idaExecutable '%1'"
-  VM-Add-To-Right-Click-Menu $toolName 'Open with IDA' "powershell.exe -WindowStyle hidden `"$script`"" "$menuIcon"
-  # Repeat for x64
-  $script = "`$idaExecutable = Get-Item '$Env:programfiles\IDA Pro *\ida64.exe' | Select-Object -Last 1; if (!`$idaExecutable) { `$idaExecutable = '$executablePath' }; & `$idaExecutable '%1'"
-  VM-Add-To-Right-Click-Menu $toolName-64 'Open with IDA (x64)' "powershell.exe -WindowStyle hidden `"$script`"" "$executablePath"
+
+  # Download ida_launcher.exe to assist with taskbar and right click option and store it in %RAW_TOOLS_DIR%
+  # ida_launcher is a custom binary that searches for the latest IDA installation and executes it
+  $launcherName = 'ida_launcher.exe'
+  $launcherSource = 'https://raw.githubusercontent.com/mandiant/vm-packages/main/ida_launcher/ida_launcher.exe'
+  $launcherPath = Join-Path ${Env:RAW_TOOLS_DIR} $launcherName
+  Write-Host "[+] Downloading '$launcherName' from '$launcherSource'"
+  try {
+    (New-Object System.Net.WebClient).DownloadFile($launcherSource, $launcherPath)
+  } catch {
```

Comment:
I think we shouldn't catch this exception, as the package should fail if we are not able to download the launcher.

I would in addition check that the binary exists after downloading it and before adding it to the registry with `VM-Assert-Path $launcherPath`